code-challenges.js

```diff
@@ -8,6 +8,24 @@ var people = [
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
   { name: "arthur dent", occupation: "radio employee" }
 ]
+
+
+const objects = (array) => {
+  let firstLetterCapitalized = array.map(value => {
+// // Now, because each name has a first and a last name (and there is no comma so it's only one object) we have to split our array in order to map through it, and capitalized both the firt and last name.
+let splitThoseNames = value.name.split(" ")
+
+ return splitThoseNames.map(value => value.charAt(0).toUpperCase()
+ + value.substring(1)).join(" ") 
+ + ` is a ${value.occupation}.`
+})
+// return the whole thing
+return firstLetterCapitalized
+}
+console.log(objects(people));
+
+
+
```

Comment:
I'd recommend that you rethink the naming of your variables and function names. They should be descriptive to explain what you are trying to accomplish. For example, you named this function "objects" which won't tell another developer anything about what you are trying to accomplish. A better function name may have been something like "descriptionMaker" or "sentanceMaker".